server/src/com/cloud/alert/AlertManagerImpl.java

```diff
@@ -767,7 +767,9 @@ public void sendAlert(AlertType alertType, long dataCenterId, Long podId, Long c
                 // set up a new alert
                 AlertVO newAlert = new AlertVO();
                 newAlert.setType(alertType.getType());
-                newAlert.setSubject(subject);
+                //do not have a seperate column for content.
+                //appending the message to the subject for now.
+                newAlert.setSubject(subject+content);
```

Comment:
@rafaelweingartner 
I understand the changes need are simple. I think that we are over exaggerating the consequence here. The worst that can happen is a db exception if the length of the sting becomes greater than 999, which has not happened until now. It dose not effect any of the critical functionality.  So i think we can fix this later in another PR. Besides this PR has been in the review board since more than a year now and i do not want to spend any more time on this. If you think this is really a critical bug and we need to fix it asap, I will be more than happy if you can lend me a hand and take this forward. 
Thanks
Bharat.